src/Keboola/StorageApi/Client.php

```diff
@@ -140,12 +142,10 @@ private function initClient()
             'backoffMaxTries' => $this->backoffMaxTries,
         ]);
 
-        if ($this->logger) {
-            $handlerStack->push(Middleware::log(
-                $this->logger,
-                new MessageFormatter("{hostname} {req_header_User-Agent} - [{ts}] \"{method} {resource} {protocol}/{version}\" {code} {res_header_Content-Length}")
-            ));
-        }
+        $handlerStack->push(Middleware::log(
```

Comment:
Tady je ještě potřeba jako třetí parametr nastavit `LogLevel::DEBUG`.

Reply:
super a to mam pred sebou napsany na papirku abych na to nezapomnel :man_facepalming: